inc/c_logging/consolelogger.h

```diff
@@ -14,6 +14,8 @@ extern "C" {
 
 #if (defined(_MSC_VER))
     void consolelogger_log_with_GetLastError(const char* file, const char* func, int line, const char* format, ...);
+#else
+    void consolelogger_log_with_GetErrorNo(const char* file, const char* func, int line, const char* format, ...);
```

Comment:
>consolelogger_log_with_GetErrorNo [](http://example.com/codeflow?start=9&length=33)

With logging v2 hopefully these gimmicks will not be needed anymore, we'd just use a log with the property we want

Reply:
That would be nice.  All in due time.